doc/sphinx/source/graph_object.rst

```diff
@@ -3,3 +3,4 @@ Graph object
 
 .. autoclass:: graphdatascience.graph.graph_object.Graph
    :members:
+   :exclude-members: __init__
```

Comment:
:sweat_smile:

Reply:
Yeah it doesn't seem like a good idea to document the constructors of the objects that are supposed to be constructed using actual procedure calls. Or factories, depending on how you view it.